inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -39,7 +39,14 @@ void LayerTestsCommon::Compare(const std::vector<std::uint8_t> &expected, const
     const auto actualBuffer = lockedMemory.as<const std::uint8_t *>();
 
     const auto &precision = actual->getTensorDesc().getPrecision();
-    const auto &size = actual->size();
+    auto resize = 1;
+    auto bathSize = 1;
+    // With dynamic batch, you need to size
+    if (configuration.count(InferenceEngine::PluginConfigParams::KEY_DYN_BATCH_ENABLED)) {
+        resize = actual->getTensorDesc().getDims()[0];
+        bathSize = resize > 1 ? resize/ 2 : 1;
+    }
+    const auto &size = (actual->size() * bathSize / resize);
```

Comment:
It's better to move `actual->size() * bathSize / resize` expression into `if` statement.

like:
`
auto &size = actual->size();
if (configuration.count(InferenceEngine::PluginConfigParams::KEY_DYN_BATCH_ENABLED)) {
         batchSize = actual->getTensorDesc().getDims()[0];
         halfBatchSize = resize > 1 ? resize/ 2 : 1;
         size = (actual->size() * halfBatchSize / batchSize)
     }
`